src/styles/variables.scss

```diff
@@ -8,3 +8,13 @@ $white: #FFFFFF;
 $offwhite: #E5E5E5;
 $rocket: #809CFF;
 $accent: #FEBCFD;
+
+/* Transitions */
+$link-transition-time: 0.35s;
+$button-transition-time: 0.6s;
+
+.animation {
```

Comment:
would this be more suited to be in `src/styles/utils.scss`?

Reply:
ah I see - it has to be here. This is probably more of a problem with the stylesheet structure - something to look into eventually maybe 🤷‍♂️